.env.example

```diff
@@ -4,7 +4,7 @@ FB_APP_SECRET=
 FB_APP_ID=
 
 #mail
-
+MAILER_SENDER_EMAIL=
```

Comment:
Should follow same name pattern, something like SMTP_EMAIL or SMTP_SENDER_EMAIL?